src/libs/polyfill/index.js

```diff
@@ -0,0 +1,18 @@
+import shouldPolyfill from './intl-polyfill';
+
+/**
+ * Polyfill the Intl API if the ICU version is old.
+ * This ensures that the currency data is consistent across platforms and browsers.
+ */
+export default function polyfill() {
+    if (!shouldPolyfill()) {
+        return;
+    }
+
+    // Just need to polyfill Intl.NumberFormat for web based platforms
+    require('@formatjs/intl-numberformat/polyfill-force');
+
+    // Load en & es Locale data
+    require('@formatjs/intl-numberformat/locale-data/en');
+    require('@formatjs/intl-numberformat/locale-data/es');
```

Comment:
Shall we create a new file `src/libs/IntlPolyfill/polyfillNumberFormat.js` for polyfilling number format and put these common imports? Or is it overkill here?